ods_ci/tests/Tests/0100__platform/0101__deploy/0102__auth_providers_and_rbac/0103__auth_crd.robot

```diff
@@ -15,18 +15,6 @@ Verify Auth Crd
     [Tags]      Smoke
     ...         Operator
     ...         RHOAIENG-18846
-    ${rc}       ${odhdashboardconfig_groups}=    Run And Return Rc And Output
-    ...    oc get odhdashboardconfig odh-dashboard-config -n ${APPLICATIONS_NAMESPACE} -o jsonpath='{.spec.groupsConfig}'
-    Should Be Equal As Integers     ${rc}       0
-    ${rc}       ${auth_cr_groups}=      Run And Return Rc And Output
-    ...                     oc get auth auth -o jsonpath='{.spec}'
-    Should Be Equal As Integers     ${rc}       0
-    ${auth_cr_groups_str}=      Remove String       ${auth_cr_groups}   [   ]
-    IF    """${odhdashboardconfig_groups}""" != """${auth_cr_groups_str}"""
-        Fail        User Groups on Auth Cr and OdhDashboardConfig DO NOT match
-    ELSE
-        Log To Console      User Groups on Auth Cr and OdhDashboardConfig match
-    END
     Get User Groups From Auth Cr And Check Rolebinding Exists       adminGroups     rolebinding     admingroup-rolebinding
     Get User Groups From Auth Cr And Check Rolebinding Exists       adminGroups     clusterrolebinding     admingroupcluster-rolebinding
     Get User Groups From Auth Cr And Check Rolebinding Exists       allowedGroups   rolebinding     allowedgroup-rolebinding
```

Comment:
In case we are still checking these, it would be good to have the Auth CR content as well as these rolebindings should exist only if the respective groups are configured in the Auth CR.

Reply:
Do you want me to discard the last commit then?

Reply:
I think just the dashboardconfig check should be removed. And we should check the content of Auth CR that it contains `{"adminGroups":"rhods-admins","allowedGroups":"system:authenticated"}` which is the default.

Reply:
I will include that

Reply:
Wait, I think that check is already being done on the Get User Groups From Auth Cr And Check Rolebinding Exists Keyword

Reply:
OK, makes sense. We could check the expected default values because now we don't check the content, just that whatever is in Auth CR is referenced in these rolebindings.
However, that will fail post-upgrade smokes I think because during upgrade testing we change the defaults and then when we run smokes after it would be failing as there will be other values than defaults in Auth CR. So I think it is fine as of now probably.

Reply:
>However, that will fail post-upgrade smokes I think because during upgrade testing we change the defaults and then when we run smokes after it would be failing as there will be other values than defaults in Auth CR.


post upgrade suite teardown should restore the default values, shouldn't it?

Reply:
@bdattoma @asanzgom yep, it should, but as of now it doesn't since we ran into issues with that. That when Smokes were run after upgrade test suite, it was failing on this.
BUT now when I am looking at it, it is probably restored:
https://github.com/red-hat-data-services/ods-ci/blob/561872afffc24caaa145987d6fa80fe6a5ea74a8/ods_ci/tests/Tests/0200__rhoai_upgrade/0203__post_upgrade.robot#L353-L352
Which [patches OdhDashboardConfig](https://github.com/red-hat-data-services/ods-ci/blob/561872afffc24caaa145987d6fa80fe6a5ea74a8/ods_ci/tests/Resources/ODS.robot#L104) back. However, now it should be patching Auth CR probably, wdyt @asanzgom ?

Reply:
indeed, it is using https://github.com/red-hat-data-services/ods-ci/blob/1c7713fce60cb5fe488df2e97447e5f0218bc466/ods_ci/tests/Resources/ODS.robot#L105 which patches the OdhDashboardConfig